src/Repository/ProductCommentRepository.php

```diff
@@ -408,4 +533,88 @@ private function getLastComment(array $criteria)
 
         return empty($comments) ? [] : $comments[0];
     }
+
+    /**
+     * @param ProductComment $productComment
+     * @param int $validate
+     *
+     * @return bool
+     */
+    public function validate($productComment, $validate = 1)
+    {
+        $success = $this->connection->executeUpdate('
+            UPDATE `' . _DB_PREFIX_ . 'product_comment` SET
+            `validate` = ' . $validate . '
+            WHERE `id_product_comment` = ' . $productComment->getId());
+
+        Hook::exec('actionObjectProductCommentValidateAfter', ['object' => $productComment]);
```

Comment:
There was no other hooks? I'm surprised only this one is in the PR body

Reply:
I see no other. It is migrated from [ProductComment.php](https://github.com/PrestaShop/productcomments/blob/dev/ProductComment.php#L354). Maybe @jolelievre knows its history because he saw it since [Version 4.0.0 version](https://github.com/PrestaShop/productcomments/pull/26)